scripts/memory_logger.py

```diff
@@ -409,7 +391,7 @@ def __init__(self, arguments, connection=None):
     if self.connection == None and self.arguments.call_back_host != None:
       self.CREATED_CONNECTION = True
       self.connection = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-      self.connection.settimeout(15)
+      self.connection.settimeout(150)
       self.connection.connect((self.arguments.call_back_host[0], int(self.arguments.call_back_host[1])))
```

Comment:
In general when you use constants, it's good to explain why. What are the first and second entries in `call_back_host` here? Also, you changed the timeout just above, why?